python/tests/unit/arcticdb/version_store/test_filtering.py

```diff
@@ -1017,9 +1017,16 @@ def test_filter_numeric_isnotin_hashing_overflow(lmdb_version_store):
 _uint64_max = np.iinfo(np.uint64).max
 
 
-@pytest.mark.parametrize("op", ("in", "not in"))
-@pytest.mark.parametrize("signed_type", (np.int8, np.int16, np.int32, np.int64))
-@pytest.mark.parametrize("uint64_in", ("df", "vals") if PANDAS_VERSION >= Version("1.2") else ("vals",))
+@pytest.mark.parametrize("op", ["in", "not in"])
+@pytest.mark.parametrize("signed_type", [np.int8, np.int16, np.int32, np.int64])
+@pytest.mark.parametrize(
+    "uint64_in",
+    ["df", "vals"]
+    if PANDAS_VERSION >= Version("1.2")
+    else [
+        "vals",
+    ],
```

Comment:
God, this black formatting is impossible to read. Could you not turn them into lists...